### PR TITLE
feat: chromealive

### PR DESCRIPTION
**Pull request by blakebyrnes**
Also convert projects to vue2 and make monorepo work

### CODE REVIEW

File: chromealive-core/index.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    const connection = this.getConnection();
+    connection.on('connected', this.onWsConnected.bind(this));
+    GlobalPool.events.on('browser-launched', this.launchApp);
+    GlobalPool.events.on('all-browsers-closed', this.closeApp);
+    GlobalPool.events.on('session-created', this.onSessionCreated);
+    FocusedWindowCorePlugin.onVisibilityChange = this.changeActiveSessions.bind(this);
+    Core.use(FocusedWindowCorePlugin);
+  }
+
+  private static onSessionCreated(event: { session: Session }): Promise<any> {
```
Comment (calebjclark) [inline chromealive-core/index.ts:61]: Can we name these heroSessions?

Comment (blakebyrnes) [reply chromealive-core/index.ts:61]: Are you proposing changing the events inside of Hero? Or is this about the destructuring of the object? I can definitely see calling it "HeroSessionObserver" unless it ends up also listening to databox sessions

Comment (calebjclark) [reply chromealive-core/index.ts:61]: I'm talking about onSessionCreated being renamed to onHeroSessionCreated.

Comment (calebjclark) [reply chromealive-core/index.ts:61]: It's confusing as to whether this is onDataboxSessionCreated or onHeroSessionCreated.

**Pull request merged**